gf-webapp-admin/src/component/rooms/Rooms.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect, useState} from "react";
+import {IRoom} from "../../model/room";
+import {RoomsApi} from "../../../autogenerated/gf-server-api/apis/RoomsApi";
+import styles from "./rooms.module.scss";
+import {Room} from "./room/Room";
+
+export const Rooms = () => {
+  const roomsApi = new RoomsApi;
+  const [rooms, setRooms] = useState<IRoom[]>([]);
```

Comment:
The component could not be initialized with empty data, it's bad practice. Use the Loader component and isInit state to handle it.

Reply:
Add loader while rooms loading